pkg/controller/hawtio/hawtio_controller.go

```diff
@@ -297,13 +306,21 @@ func (r *ReconcileHawtio) Reconcile(request reconcile.Request) (reconcile.Result
 	}
 
 	if isOpenShift4 {
-		// Check whether client certificate secret exists
+		//Check whether client certificate secret exists
 		_, err := r.coreClient.Secrets(request.Namespace).Get(ctx, request.Name+"-tls-proxying", metav1.GetOptions{})
 		if err != nil {
 			if errors.IsNotFound(err) {
-				reqLogger.Info("Client certificate secret must be created", "secret", request.Name+"-tls-proxying")
-				// Let's poll for the client certificate secret to be created
-				return reconcile.Result{Requeue: true, RequeueAfter: 5 * time.Second}, nil
+				secretName := request.Name + "-tls-proxying"
+				reqLogger.Info("Client certificate secret must be created", "secret", secretName)
+				reqLogger.Info("Creating the certificate")
+
+				caSecret, _ := r.coreClient.Secrets("openshift-service-ca").Get(ctx, "signing-key", metav1.GetOptions{})
```

Comment:
I think we have to update the role permissions in the deploy resources and in the OLM bundle, so that the operator service account are granted permission to get the service signing CA secret.